frontend/src/routes/(frontend)/explore/index.tsx

```diff
@@ -47,11 +46,11 @@ export default component$(() => {
 
 export const requestLoader = loader$(async ({ request }) => {
 	// Manually parse the JSON to ensure that Qwik finds the resulting objects serializable.
-	return JSON.parse(JSON.stringify(request)) as RequestContext
+	return JSON.parse(JSON.stringify(request)) as Request
```

Comment:
is this still required? Could we open an issue against Qwik?

Reply:
I am quite sure that this is a kind of known limitation and it's going to stay like this since the request object has a bunch of stuff that makes it non serializable

with the `JSON.parse(JSON.stringify(` we're stripping the non-serializable stuff out, kind of like this for example:
![Screenshot 2023-02-17 at 17 29 38](https://user-images.githubusercontent.com/61631103/219724370-6bac55f9-690a-44f2-9ece-a98e6a4acfa7.png)

(I could not find a better official qwik link, but this kind of says that you need to keep things serializable: [Writing applications with serializability in mind](https://qwik.builder.io/docs/concepts/resumable/#writing-applications-with-serializability-in-mind))

Reply:
mh.... maybe we could ask them if they could implement the non-serializable props dropping... 🤔 

although I feel like that could lead to confusion for developers (if they start missing props that they were expecting)

Reply:
I've opened an issue in their github repo: https://github.com/BuilderIO/qwik/issues/3019 👍